src/utils/api.ts

```diff
@@ -2,14 +2,14 @@ import axios from "axios";
 
 const api = axios.create({
   baseURL: `${process.env.NEXT_PUBLIC_API_BASE_URL}`, // 백엔드 주소
-  withCredentials: true, // 쿠키 자동 전송 활성화
+  // withCredentials: true, // 쿠키 자동 전송 활성화설정인데 cors에러를 반환해서 주석처리 해놨습니다.
 });
 
 api.interceptors.request.use(
```

Comment:
```suggestion
apiClient.interceptors.request.use(
```
따로 만들 필요없고 apiClient 파일에 인터셉터 추가하시면 될 것 같아용!